### DISCUSSION TITLE
WiX 4 - Cannot find insignia.exe in WiX 4

### DISCUSSION THREAD

**Discussion by garevaul** [wixtoolset/issues#7375] [Questions] [answered]
Hi,

I cannot find insignia.exe in WiX 4.

I was using insignia.exe along with SignTool.exe to sign my final bootstrapper project's EXE file.

I was using Insignia.exe, because in Wix3 the EXE installer did not work when I used ONLY SignTool.exe to sign the EXE.

Regards

**Comment by barnson** [comment, marked as answer]
https://wixtoolset.org/docs/tools/signing/

**Reply by BMurri** [reply]
And the Reference: https://wixtoolset.org/docs/tools/wixexe/#burnsigning

**Reply by garevaul** [reply]
Thanx @barnson. I incorporated 'Wix burn' now, and signing works excellent now.
